.github/workflows/pr-autofix.yml

```diff
@@ -20,13 +20,13 @@ jobs:
 
           then
 
-          echo "::set-output name=changed::true"
+          echo "name=changed::true" >> $GITHUB_STATE
```

Comment:
**thought:** I think this should be `$GITHUB_OUTPUT` so it can be used by the next step, but I'm not fully sure if STATE would work here as well.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
Gah!  You're right!!! Thanks for catching that @jandrade